tests/atomizer.js

```diff
@@ -899,7 +899,11 @@ describe('Atomizer()', function () {
         it ('properly handles classnames with optional arguments', function () {
             var atomizer = new Atomizer();
             var config = {
-                classNames: ['Skew(90deg)', 'Skew(90deg,45deg)', 'Bdsp(1em)', 'Bdsp(2em,33%)']
+                custom: {
+                    'sc': 'small-caps',
```

Comment:
`sc` is already defined as a predefined argument for `font-variant`, so you wouldn't want to confuse things by having a custom value with the same name.